packages/@webex/plugin-meetings/src/multistream/remoteMediaManager.ts

```diff
@@ -686,16 +752,54 @@ export class RemoteMediaManager extends EventsScope {
         );
       }
     }
-    // todo: screenshare (SPARK-377812)
 
     this.mediaRequestManagers.video.commit();
   }
 
+  private updateScreenShareVideoRemoteMediaObject() {
+    if (this.config.screenShare.video) {
+      this.invalidateCurrentRemoteMedia({
+        audio: false,
+        video: false,
+        screenShareAudio: false,
+        screenShareVideo: true,
+        commit: false,
+      });
+
+      this.media.screenShare.video = undefined;
+
+      if (this.currentLayout?.screenShareVideo) {
+        if (this.config.screenShare.video) {
```

Comment:
Didn't you already check for this when you loaded the config?

Reply:
done

Reply:
looking at this more I think the check at the beginning of the function is also redundant so I'll remove that too